common/common.go

```diff
@@ -0,0 +1,28 @@
+package common
```

Comment:
Could we have a different name for this package? ``types`` or ``api``, maybe? :thinking:

Reply:
done